.github/workflows/ci.yaml

```diff
@@ -34,9 +34,6 @@ jobs:
         with:
           path: ~/.sbt
           key: sbt-${{ hashFiles('**/build.sbt') }}
-      - name: Check formatting
```

Comment:
Do we have formatting checks elsewhere?

Reply:
Not that I know of. How do get it working on 2.13? It blew up when I just removed the `if`.

Reply:
Could we try changing the `'2.12.'` to `'2.13.'`? IIRC, the `if` is there so that we only run checks once per build, instead of for every matrix permutation.

Reply:
Okay, I'll try that, though it doesn't explain why it failed when I removed the `if` check.

Reply:
@amohrland we aren't building with Java 1.8 at all anymore. The matrix has a size of one. So changing to 2.13 won't do anything.

Reply:
Maybe we can try changing it to match what we're building with then? 2.13 and `adopt@1.11.0-9`, it looks like.

Reply:
If it doesn't work out for some reason, then let's circle back to it later. Formatting checks are just a nice-to-have. The rest of the PR looks great, and I think we should move ahead with it.